.travis.yml

```diff
@@ -11,21 +11,19 @@ python:
   - 'pypy3'
 
 env:
-  - DJANGO_VERSION=master
```

Comment:
django master / 3.x requires Python 3.6 since we'd exclude 2.7 to 3.5 and both pypy versions so drop it here and add it back below since it requires xenial for SQLite 3.8.3